packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -451,6 +451,9 @@ function BackgroundSizeControls( {
 	const positionValue =
 		style?.background?.backgroundPosition ||
 		inheritedValue?.background?.backgroundPosition;
+	const attachmentValue = style?.background?.backgroundAttachment || {
+		...inheritedValue?.background?.backgroundAttachment,
+	};
```

Comment:
```suggestion
	const attachmentValue =
		style?.background?.backgroundAttachment ||
		inheritedValue?.background?.backgroundAttachment;
```

Am I correct in thinking the `backgroundAttachment` is a simple string and the inherited value should be used directly?

If I apply the suggestion locally, it fixes the control not displaying the inherited value from theme.json.

Should we also be updating the theme.json schema to indicate the expected type for `backgroundAttachment`, or is that premature?

Reply:
You're right on all counts. I've pushed a fix. 👍🏻